client/dist/index.html

```diff
@@ -2,6 +2,7 @@
 <html>
 <head>
 	<title>AirBnB</title>
+	<link rel="stylesheet" href= "https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
```

Comment:
You have bootstrap installed via npm and a link to the CDN. Which one do you plan on using?

Reply:
Good catch--I have decided on using the npm package instead!